db/migrate/20200626165535_add_poo_status_to_institutions.rb

```diff
@@ -0,0 +1,6 @@
+class AddPooStatusToInstitutions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :institutions, :poo_status, :string
```

Comment:
Just confirming - no default value needed?

Reply:
correct, the value should be provided in the WEAMS csv and is inserted into institutions during use of https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/models/weam.rb#L20 which is updated in https://github.com/department-of-veterans-affairs/gibct-data-service/pull/676